src/vm/gcenv.ee.cpp

```diff
@@ -1349,3 +1349,39 @@ void GCToEEInterface::HandleFatalError(unsigned int exitCode)
 {
     EEPOLICY_HANDLE_FATAL_ERROR(exitCode);
 }
+
+bool GCToEEInterface::ShouldFinalizeObjectForUnload(AppDomain* pDomain, Object* obj)
+{
+    MethodTable *pMT = obj->GetMethodTable();
```

Comment:
This can just `return true` in CoreCLR - it should be never actually called. We do not have AppDomains in CoreCLR.